releasenotes/notes/ansible-requirements-bump-89313038efba83b3.yaml

```diff
@@ -0,0 +1,10 @@
+---
+features:
+  - |
+    Ansible-Galaxy dependencies have been bumped to the latest available
+    versions. This includes:
+
+    pulp.squeezer - 0.0.13 -> 0.1.0
+    ansible-modules-hashivault - 1.3.1 -> 1.3.4
+    geerlingguy.pip - 2.2.0 -> 3.1.0
+    monolithprojects.github_actions_runner - 1.18.5 -> 1.25.1
```

Comment:
I think this won't format correctly. You need to write a bullet list.

Reply:
Thanks, fixed